README.md

```diff
@@ -50,15 +44,15 @@ of things. Instead of replacing sane data management that should be easy,
 
 ## Quick Start Guide
 
-First, install `urql` and `graphql`, which is a peer dependency:
+First install `urql` and `graphql` (which is a peer dependency, justification for this is explained [here](docs/getting-started.md)):
```

Comment:
I don't think the link is necessary at this point :) This is just the quick start guide and we already link to "Getting Started" later